src/main/java/com/google/devtools/build/lib/analysis/config/CoreOptions.java

```diff
@@ -433,6 +433,17 @@ public ExecConfigurationDistinguisherSchemeConverter() {
               + ".runfiles/wsname/external/repo (in addition to .runfiles/repo).")
   public boolean legacyExternalRunfiles;
 
+  @Option(
+      name = "incompatible_always_include_files_in_data",
+      defaultValue = "true",
```

Comment:
I'd need "false" here, if I want to import. I can flip it to true as soon as it's released internally.

Reply:
Changed